include/psa/crypto.h

```diff
@@ -738,8 +738,9 @@ psa_status_t psa_import_key(const psa_key_attributes_t *attributes,
  *   `PSA_ECC_CURVE_CURVEXXX`), and in big-endian order for Weierstrass
  *   curves (curve types `PSA_ECC_CURVE_SECTXXX`, `PSA_ECC_CURVE_SECPXXX`
  *   and `PSA_ECC_CURVE_BRAINPOOL_PXXX`).
- *   This is the content of the `privateKey` field of the `ECPrivateKey`
- *   format defined by RFC 5915.
+ *   For Weierstrass curves, this is the content of the `privateKey` field of
+ *   the `ECPrivateKey` format defined by RFC 5915.  For Montgomery curves,
+ *   the format is defined by RFC 7748, and output is masked according to §5.
```

Comment:
About 8d5617dd2db2e868e07053304ef2aca130f1c5f7: Please limit the summary line of git commit messages to 72 characters if at all possible. It isn't an absolute rule, especially if you need to fit a long function name, but we strongly prefer it because some interfaces (such as GitHub) truncate the first line.

For example:

> Document masking  of Montgomery private keys in psa_export_key

> Follow the PSA Crypto specification which was updated between 1.0 beta 3
> and 1.0.0.

> Add corresponding test cases.

Also applies to 5410dfa25ba35285f7eff650066648f2a6fb75b3